prober/tcp.go

```diff
@@ -129,10 +129,14 @@ func ProbeTCP(ctx context.Context, target string, module config.Module, registry
 	}
 	if module.TCP.TLS {
 		state := conn.(*tls.Conn).ConnectionState()
-		registry.MustRegister(probeSSLEarliestCertExpiry, probeSSLLatestVerifiedChainExpiry, probeTLSVersion)
+		registry.MustRegister(probeSSLEarliestCertExpiry, probeTLSVersion)
 		probeSSLEarliestCertExpiry.Set(float64(getEarliestCertExpiry(&state).Unix()))
-		probeSSLLatestVerifiedChainExpiry.Set(float64(getLatestVerifiedChainExpiry(&state).Unix()))
 		probeTLSVersion.WithLabelValues(getTLSVersion(&state)).Set(1)
+
+		if lastChainExpiry, ok := getLastChainExpiry(&state); ok {
+			registry.MustRegister(probeSSLLastChainExpiryTimestampSeconds)
```

Comment:
Let's keep this consistent with the other tls metric, both in terms of registration and what happens when there's no certs.